Mathlib/GroupTheory/Perm/Basic.lean

```diff
@@ -378,7 +378,6 @@ theorem ofSubtype_subtypePerm {f : Perm α} (h₁ : ∀ x, p x ↔ p (f x)) (h
     by_cases hx : p x
     · exact (subtypePerm f h₁).extendDomain_apply_subtype _ hx
     · rw [ofSubtype, MonoidHom.coe_mk]
-      -- Porting note: added `dsimp`
```

Comment:
Is the dsimp still needed? This porting note reads like the `rw` used to work in Lean 3, but now needs the`dsimp` before it is successful: is this still true?

Reply:
I don't know what happened here. This can all be one `rw`, but somehow didn't end up in the patch :shrug: